### PR TITLE
nginx 1.19.6

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by bayandin** [comment]
On 10.14:
```
/private/tmp/passenger-20201216-39420-1knyo4d/passenger-6.0.7/src/ruby_native_extension/extconf.rb:28:in `open': No such file or directory @ dir_initialize - /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/include/ruby-2.3.0 (Errno::ENOENT)
	from /private/tmp/passenger-20201216-39420-1knyo4d/passenger-6.0.7/src/ruby_native_extension/extconf.rb:28:in `entries'
	from /private/tmp/passenger-20201216-39420-1knyo4d/passenger-6.0.7/src/ruby_native_extension/extconf.rb:28:in `<main>'
rake aborted!
```

**Comment by chenrui333** [comment]
It looks like that I could not reproduce in my local.

**Comment by chenrui333** [comment]
my ruby is 

```
$ /usr/local/opt/ruby/bin/ruby --version
ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-darwin18]
```

File: Formula/passenger.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ class Passenger < Formula
   depends_on "nginx" => [:build, :test]
   depends_on "openssl@1.1"
   depends_on "pcre"
+  depends_on "ruby"
```
Comment (SeekingMeaning) [inline Formula/passenger.rb:21]:
```suggestion
  uses_from_macos "ruby", since: :catalina
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Nice!

**Pull request closed without merging**